scripts/set-react-version.mjs

```diff
@@ -67,7 +67,7 @@ async function checkEnvironment() {
   // See https://nodejs.org/en/blog/release/v18.14.0.
 
   const [major, minor] = process.versions.node.split(".");
-  const nodeVersion = Number.parseInt(major) * 100 + Number.parseInt(minor);
+  const nodeVersion = Number(major) * 100 + Number(minor);
```

Comment:
just silly me asking questions: `.parseInt()` was redundant and that's why you took it off, correct?

Reply:
Yeah, you really only need it if you use a different base than 10.